packages/editor/src/commands/ScaleCommand.ts

```diff
@@ -137,6 +137,7 @@ function updateScale(command: ScaleCommandParams, isUndo: boolean): void {
 
       transformComponent.scale.copy(tempVector)
     }
+    if (obj3d.userData.updateTransform) obj3d.userData.updateTransform()
```

Comment:
This can be moved to `BoxColliderNodeEditor.tsx` and can be invoked by watching the `Selection` state.

Reply:
> 

Hmm.. in the `BoxColliderNodeEditor.tsx`, we could not detect scale transform changes. so I am still going to keep the above code. please let me know the suggestions.

Reply:
```
const selectionState = useSelectionState()
useEffect(() => {
    if (selectionState.propertyName.value === 'scale') {
      console.log('update the box collider')
    }
  }, [selectionState.propertyName.value])
```

This can be the solution.